.github/pull_request_template.yaml

```
@@ -25,7 +25,6 @@ Applicable spec: <link>
 - [ ] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
 - [ ] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
 - [ ] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
-- [ ] The documentation is generated using `src-docs`
 - [ ] The documentation for charmhub is updated
 - [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `senior-review-required`, `documentation`)
 - [ ] The `docs/changelog.md` is updated with user-relevant changes.
```

Nitpick: Since there are three different components in this repo, maybe the checklist item should say something like:
```suggestion
- [ ] The appropriate `docs/changelog.md` is updated with user-relevant changes.
```